views/default/input/userpicker.php

```diff
@@ -51,7 +51,9 @@
 	</ul>
 </div>
 <script>
-require(['elgg/UserPicker'], function (UserPicker) {
-	UserPicker.setup('.elgg-user-picker[data-name="<?php echo $name ?>"]');
+elgg_defer(function () {
+	require(['elgg/UserPicker'], function (UserPicker) {
```

Comment:
This nesting is unfortunate... Is there a way we could just do this with `require` to get people "thinking in amd"?